qontract_development_cli/templates/compose.yml.j2

```diff
@@ -47,6 +47,12 @@ services:
 {% endif %}
     - gitlab_pr_submitter_queue_url={{ env.settings.gitlab_pr_submitter_queue_url }}
     - LOG_LEVEL={{ profile.settings.log_level.upper() }}
+{% if  profile.settings.localstack %}
+    - AWS_ENDPOINT_URL=http://localstack:4566
+    - AWS_ACCESS_KEY_ID=fake
+    - AWS_SECRET_ACCESS_KEY=fake
```

Comment:
better to set to `test` to avoid issues

https://docs.localstack.cloud/user-guide/integrations/aws-cli/#configuring-an-endpoint-url

>To enable the creation of pre-signed URLs for S3 buckets, please set both AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY to the value “test.” Our pre-signed URL signature verification algorithm validates the pre-signed URL and its expiration.

Reply:
Changed in the last commit